### DISCUSSION TITLE
핸들러와 스토리지 엔진은 다른 개념 아닌가요??

### DISCUSSION THREAD

**Discussion by emost22** [Well-Founded-Confidence/RealMySQL#21] [Q&A] [answered]
4.1.6.5 핸들러 내용 중
핸들러는 결국 스토리지 엔진을 의미하며,
MyISAM 테이블을 조작하는 경우에 핸들러는 MyISAM 스토리지 엔진이 되고
InnoDB 테이블을 조작하는 경우에 핸들러가 InnoDB 스토리지 엔진이 된다.

4.1.4 플러그인 스토리지 엔진 모델에도 핸들러가 나오는데
MySQL 엔진이 각 스토리지 엔진에게 데이터를 읽어오거나 저장하도록 명령하려면 반드시 핸들러를 이용해야 한다. 라고 소개됩니다.

결론은 핸들러는 스토리지 엔진을 의미하는게 아니고, 스토리지 엔진을 사용한다는 의미로 바꾸는 게 맞지 않을까요?

**Comment by menthamin** [comment, marked as answer]
핸들러 == 스토리지엔진
- 
Win: 핸들러 != 스토리지 엔진
- 핸들러가 엔진에 맞게 잘 바꿔준다.

MySQL 엔진 -> 핸들러 -> 스토리지 엔진
